src/condor_credd/condor_credmon_oauth/examples/config/condor/40-oauth-credmon.conf

```diff
@@ -86,3 +96,13 @@ use feature : OAUTH
 # SEC_CREDENTIAL_DIRECTORY_OAUTH = /var/lib/condor/oauth_credentials
 # CREDMON_OAUTH_LOG = $(LOG)/CredMonOAuthLog
 # CREDMON_OAUTH = /usr/sbin/condor_credmon_oauth
+
+## These settings only apply to credentials fetched by Vault
+## Set the minimum time in seconds that access tokens must have
+## before they expire when they are fetched by credmon. It must
+## be set to be less than the expiration time assigned by the token
+## issuer.
```

Comment:
We can detect the expiration time (delay/duration, presumably) assigned by the issuer, right?  Should this be being computed, rather than configured?

Reply:
In theory, the access tokens that we get back from Vault should be opaque, so we shouldn't be inspecting them for their expiration dates. I don't think there's a standard for how to query issuers for expiration dates, and the issuer could be doing weird things like giving user A tokens that only live for 10 minutes and user B tokens that live for a week 🤷.

Reply:
Understood.  Maybe an extra sentence in the comment?  Something like "We can't inspect Vault's access tokens for their expiration date, so we need to be told how often to renew them."

Reply:
Note added in 9d5065992d340f0697d9d60959a16d4832d184cf